app/assets/json/translations/locale-en.json

```diff
@@ -653,7 +656,8 @@
             "PICK_UP_AN_APPLICATION": "Pick up an application",
             "SEND_BY_US_MAIL": "Send Application by US Mail",
             "SUBMIT_A_PAPER_APPLICATION": "Submit a Paper Application",
-            "VISIT_A_HOUSING_COUNSELOR": "Visit a local housing counselor for help with your application."
+            "VISIT_A_HOUSING_COUNSELOR": "Visit a local housing counselor for help with your application.",
+            "FULFILL_ELIGIBITY_REQUIREMENTS": "Make sure you fulfill the <a href='https://sfmohcd.org/homebuyer-program-eligibility'>eligibility requirements</a> for this listing before you begin."
```

Comment:
minor nitpick, it looks like elsewhere in locales-en we pass in urls as a variable rather than hardcoding it. 
See e.g. NAV.GET_FEEDBACK